server/server.go

```diff
@@ -330,93 +325,75 @@ func (s *Server) newBuilders(logger chronograf.Logger) builders {
 }
 
 // Serve starts and runs the chronograf server
-func (s *Server) Serve(ctx context.Context) error {
+func (s *Server) Serve(ctx context.Context) {
 	logger := clog.New(clog.ParseLevel(s.LogLevel))
-	_, err := NewCustomLinks(s.CustomLinks)
+	customLinks, err := NewCustomLinks(s.CustomLinks)
 	if err != nil {
 		logger.
 			WithField("component", "server").
 			WithField("CustomLink", "invalid").
 			Error(err)
-		return err
```

Comment:
why do we not want to return the error here?

Reply:
It was causing a double logging event. since this function only gets called from main and it logs before returning. I figured rather than pull the logger into main.go and log the error there, i'd just let it log them inside this function and not worry about an error coming back in main

Reply:
I suppose it could be put back in because i think `log.fatal` exits non-zero and that was run on error in main